aiopg/utils.py

```diff
@@ -193,37 +197,97 @@ def __aexit__(self, exc_type, exc_val, exc_tb):
 class _PoolCursorContextManager:
```

Comment:
after looking at this more it wouldn't help because we need different behavior between enter and aenter